GEM_VERSION

```diff
@@ -1 +1 @@
-2.3.16
+2.3.17
```

Comment:
I think there is no need to change the version here. I would keep it the same. What do you think? 

Once we have this merged with `2-3-stable` then we can simply run `bundle update spree_avatax_certified` in mejuri-web and that would be enought for mejuri-web to use latest commit.

Reply:
Hi @sebastiandl I didn't know it was not necessary to change the version, I just switched it backed and updated the revision only and it's working 🙌

Reply:
Great! thanks!